xapian-applications/omega/query.cc

```diff
@@ -1547,7 +1563,6 @@ eval(const string &fmt, const vector<string> &param)
 			ket += '>';
 		    }
 		}
-
```

Comment:
As well as being good practice to review the diff before submitting a patch, for similar reasons it's a good idea to review the combined diff (i.e. diff against the branch point) when amending a patch.  

Assuming the branch was made from `master` at some point, then this will diff against the common ancestor of the current branch and `master` (NB **three** dots):

```
git diff master...
```